addons/spectator/stringtable.xml

```diff
@@ -3,195 +3,253 @@
         </Key>
-
         <!-- Interface strings -->
         <Key ID="STR_ACE_Spectator_HelpTitle">
             <English>Spectator Controls</English>
+            <Polish>Sterowanie obserwatorem</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_ViewFree">
             <English>Free</English>
+            <Polish>Wolna</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_ViewInternal">
             <English>Internal</English>
+            <Polish>Wewnętrzna</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_ViewExternal">
             <English>External</English>
+            <Polish>Zewnętrzna</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_VisionNormal">
             <English>Normal</English>
+            <Polish>Normalna</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_VisionNight">
             <English>Night</English>
+            <Polish>Noc</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_VisionThermal">
             <English>Thermal</English>
+            <Polish>Termo</Polish>
         </Key>
-
         <!-- Controls -->
         <Key ID="STR_ACE_Spectator_freeCamControls">
             <English>Free Camera Controls</English>
+            <Polish>Wolne sterowanie kamerą</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamForward">
             <English>Camera Forward</English>
+            <Polish>Kamera naprzód</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamBackward">
             <English>Camera Backward</English>
+            <Polish>Kamera w tył</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamLeft">
             <English>Camera Left</English>
+            <Polish>Kamera w lewo</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamRight">
             <English>Camera Right</English>
+            <Polish>Kamera w prawo</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamUp">
             <English>Camera Up</English>
+            <Polish>Kamera w górę</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamDown">
             <English>Camera Down</English>
+            <Polish>Kamera w dół</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamPan">
             <English>Pan Camera</English>
+            <Polish>Panoramowanie</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamDolly">
             <English>Dolly Camera</English>
+            <Polish>Płynna kamera</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamLock">
             <English>Lock Camera to Target</English>
+            <Polish>Zablokuj kamerę na celu</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamZoom">
             <English>Zoom In/Out</English>
+            <Polish>Zoom +/-</Polish>
         </Key>
         <Key ID="STR_ACE_Spectator_freeCamSpeed">
             <English>Speed Up/Down</English>
+            <Polish>Prędkość +/-</Polish>
```

Comment:
Oooh, I like the use of +/- here. Could you also change the English strings to use that? :smile: